pvlib/irradiance.py

```diff
@@ -2186,6 +2186,94 @@ def erbs(ghi, zenith, datetime_or_doy, min_cos_zenith=0.065, max_zenith=87):
     return data
 
 
+def boland(ghi, zenith, datetime_or_doy, min_cos_zenith=0.065, max_zenith=87):
+    r"""
+    Estimate DNI and DHI from GHI using the Boland clearness index model.
+
+    The Boland model [1]_, [2]_ estimates the diffuse fraction, DF, from global
+    horizontal irradiance, GHI, through an empirical relationship between DF
+    and the ratio of GHI to extraterrestrial irradiance or clearness index, kt.
+
+    .. math::
+
+        \mathit{DF} = \frac{1}{1 + \exp\left(-5 + 8.6 k_t\right)}
+
+    where :math:`k_t` is the clearness index.
+
+    Parameters
+    ----------
+    ghi: numeric
```

Comment:
Remind me, does `numeric` include `Series`?

Reply:
TL;DR: Yes, if the `pd.Series` is an array of a numeric type like float, integer, boolean, or complex, otherwise no. _EG_: a pandas series of strings would _not_ be numeric.

I can't find where "numeric" is defined except in NumPy here:
> There are 5 basic numerical types representing booleans (`bool`), integers (`int`), unsigned integers (`uint`) floating point (`float`) and complex.

and in [pandas `to_numeric()`](https://pandas.pydata.org/docs/reference/api/pandas.to_numeric.html).

The "numeric" data type is used extensively in pvlib and I believe its use is consistent with the NumPy & pandas definition, and I don't believe it is constrained to either scalars or arrays.